config/fxdata/objects.cfg

```diff
@@ -46,6 +46,12 @@ Properties =
 ;function that should be executed to update the object 
 ;UPDATE_DUNGEON_HEART,UPDATE_CALL_TO_ARMS,UPDATE_ARMOUR,UPDATE_OBJECT_SCALE,UPDATE_POWER_SIGHT,UPDATE_POWER_LIGHTNING
 UpdateFunction = 
+;Initial state, only used for chickens
+InitialState = 0
+;randomizes the frame at wich each anim starts, so multiple objects don't all make the same movement at same time
+RandomizeStartframe = 0
```

Comment:
Start Frame is 2 words, so it should be `RandomizeStartFrame`

Reply:
But better yet, in trapdoor.cfg we have the exact same field, but named `RandomStartFrame`,... I believe it would make sense to name both the field in the data and the field for the users the same for both traps and objects,...